src/corona_hakab_model/manager.py

```diff
@@ -76,7 +76,7 @@ def __init__(
         self.update_matrix_manager = update_matrix.UpdateMatrixManager(self)
         self.infection_manager = infection.InfectionManager(self)
         self.healthcare_manager = healthcare.HealthcareManager(self)
-        self.medical_state_manager = MedicalStateManager(self)
+        self.medical_state_manager = MedicalStateManager(self.medical_machine)
```

Comment:
We want the MedicatStateManager to be aware of objects held by the SimulationManager, like the agents list and contagiousness vector.

Reply:
Currently he has no use for the manager. If such a connection would occur in the future it should be considered to use a different architecture and not have medial state machien have manager as a field.